### ISSUE TITLE
Missing Parenthesis V104

### ISSUE THREAD

**Issue by juliusbsimon** [bostrowski/Pretius-APEX-Enhanced-Lov-Item#21]
Version 1.03 works fine but installing 1.04 on Apex 18.2 results in a missing right parenthesis error no matter what configuration is used.

**Comment by zhudock** [comment]
I received this error as well, but only when "Use Custom Search Logic" is enabled. I had to look at the APEX Debug Logs, and noticed that the SQL query being assembled by the plugin was invalid. It was appending my custom filter directly after `1=1` rather than including `and (my custom filter)`

e.g. this
```
select * 
 from products
where 1 = 1 upper(product_code) like upper(:search_string || '%') or upc like upper(:search_string || '%') 
```

instead of
```
select * 
 from products
where 1 = 1 
    and (upper(product_code) like upper(:search_string || '%')
      or  upc like upper(:search_string || '%'))
```

**Comment by bostrowski** [comment]
@zhudock 
yes, this is maintained by the plugin help text for the attribute. 

I've updated the demo application with the valid SQL query. Help Texts are most important and sometimes I might forget to update demo app ;-) 

I'm glad you have found help via help text. 

I assume the issue can be closed @zhudock?

**Comment by zhudock** [comment]
I wasn't the original submitter of the issue, but from my standpoint, i feel the issue can be closed
